From 0ea00e99d28459bddfa227189ba9462678f9f16f Mon Sep 17 00:00:00 2001
From: Herbert Xu <herbert@gondor.apana.org.au>
Date: Mon, 1 Sep 2014 13:10:40 -0400
Subject: [redhat] Kconfig: Enable DRBG by default

Message-id: <E1XORNY-00089l-Aj@gondolin.me.apana.org.au>
Patchwork-id: 90075
O-Subject: [PATCH 19/19] [crypto] drbg: Enable DRBG by default
Bugzilla: 1011562
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: Rafael Aquini <aquini@redhat.com>

RHEL7 Bugzilla #1011562

Message-id: <E1X5ato-0007eF-U6@gondolin.me.apana.org.au>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/CONFIG_CRYPTO_DRBG b/redhat/configs/generic/CONFIG_CRYPTO_DRBG
new file mode 100644
index 0000000..93d1606
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_CRYPTO_DRBG
@@ -0,0 +1 @@
+CONFIG_CRYPTO_DRBG=m
diff --git a/redhat/configs/generic/CONFIG_CRYPTO_DRBG_CTR b/redhat/configs/generic/CONFIG_CRYPTO_DRBG_CTR
new file mode 100644
index 0000000..1c292fe
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_CRYPTO_DRBG_CTR
@@ -0,0 +1 @@
+CONFIG_CRYPTO_DRBG_CTR=y
diff --git a/redhat/configs/generic/CONFIG_CRYPTO_DRBG_HASH b/redhat/configs/generic/CONFIG_CRYPTO_DRBG_HASH
new file mode 100644
index 0000000..2ec6b20
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_CRYPTO_DRBG_HASH
@@ -0,0 +1 @@
+CONFIG_CRYPTO_DRBG_HASH=y
diff --git a/redhat/configs/generic/CONFIG_CRYPTO_DRBG_HMAC b/redhat/configs/generic/CONFIG_CRYPTO_DRBG_HMAC
new file mode 100644
index 0000000..d61f043
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_CRYPTO_DRBG_HMAC
@@ -0,0 +1 @@
+CONFIG_CRYPTO_DRBG_HMAC=y
diff --git a/redhat/configs/generic/CONFIG_CRYPTO_DRBG_MENU b/redhat/configs/generic/CONFIG_CRYPTO_DRBG_MENU
new file mode 100644
index 0000000..cf41486
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_CRYPTO_DRBG_MENU
@@ -0,0 +1 @@
+CONFIG_CRYPTO_DRBG_MENU=m
-- 
1.7.1